csharp/ql/src/change-notes/2025-03-10-unknown-type-matching.md

```diff
@@ -0,0 +1,4 @@
+---
+category: minorAnalysis
+---
+* Increase query precision for `cs/useless-assignment-to-local` and `cs/constant-condition` when *unknown* types are involved (mostly relevant for build mode none databases).
```

Comment:
How do we reference buildess DBs, should it be `build-mode: none`?

Reply:
I will change it to that.
Maybe the abbreviation BMN is also acceptable? (What do you think @coadaflorin? )